Tasks/AzureResourceGroupDeploymentV2/task.loc.json

```diff
@@ -14,7 +15,7 @@
   "version": {
     "Major": 2,
     "Minor": 145,
```

Comment:
do you want to update the minor version to 147

Reply:
Updated to `147` instead